content-src/asrouter/docs/targeting-attributes.md

```diff
@@ -210,6 +212,59 @@ localeLanguageCode == "en"
 declare const localeLanguageCode: string;
 ```
 
+### `needsUpdate`
```

Comment:
i think this was copy pasted by accident, `needsUpdate` and `usesFirefoxSync` is here twice now